operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -82,15 +82,24 @@ Release dates are currently communicated in the following ways. Further iteratio
 
 Release Manager is able to look at the version we have deployed to Mattermost Cloud. Also, when we are hooked up to split.io, there will be a dashboard where you can see active and historic feature flags. Any feature that has its flag removed would be included on any Self-Managed releases past the point after that removal was merged. More details on feature flags: https://developers.mattermost.com/contribute/server/feature-flags/.
 
+## Cloud Release Branch Processes
+
+**Process for merging the ``master`` branch into the ``cloud`` branch**
+ - The Cloud team opens PRs in webapp, server, redux, and enterprise branches to merge the ``master`` branch into the ``cloud`` branch ([example PR](https://github.com/mattermost/mattermost-server/pull/16838)).
+ - Devs should be aware of the dates when the ``master`` branch is merged into the ``cloud`` branch in order to be mindful of avoiding having incomplete features in a Cloud release and to be mindful of bug fixes that we may want to include in a release.
+
+**Process for cutting the release branch for Self-Managed releases based off of Cloud releases**
```

Comment:
@metanerd My goal is to document this process for visibility for devs who may not know how it works yet. Let me know if any steps or details should be added below.

Reply:
Looks good to me, thank you Amy!